examples/ThemeMaker.py

```diff
@@ -0,0 +1,271 @@
+import tkinter
+import customtkinter
+from tkinter.colorchooser import askcolor
+from tkinter import filedialog, messagebox
+import json
+
+#Main Window
+WIDTH = 390
+HEIGHT = 350
+customtkinter.set_appearance_mode("Dark")
+customtkinter.set_default_color_theme("blue")
+self=customtkinter.CTk()

```

Comment:
Can you structure the program into a class? Don't just use `self` when its not actually inside a class.

Reply:
Changed the self to root_tk

Reply:
@TomSchimansky Done, changed the whole program structure to class